digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -70,6 +70,9 @@ private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBu
         }
         else {
             imageName = baseImageName;
+            if (dockerConfig.has("pull-always")) {
```

Comment:
Thanks!

Regarding the default value, IMO `false` is safer and more expected behaviour. If the behaviour of Docker image changes implicitly with `true` by default, it may surprise users and cause a trouble. Also,  the backward compatibility is kept for now with `false` by default.

We can discuss that again and change the default behaviour on another ticket if needed in the future.